tests/Mockery/NamedMockTest.php

```diff
@@ -65,4 +65,22 @@ public function itCreatesConcreteMethodImplementationWithReturnType()
         );
         $this->assertTrue($gardener->water($cactus));
     }
+
+    /** 
+     * @test 
+     * @requires PHP 7.0.0
+     */
+    public function it_gracefully_handles_namespacing()
+    {
+        require_once __DIR__.'/DummyClasses/NamespacedReturnTypes.php';
```

Comment:
This wouldn't be necessary if you had an `autoload-dev` namespace in `composer.json`, which should be preferred, since having such an definition will preclude the use of this redundant pattern throughout your tests.

Reply:
👍